app/_src/guides/restrict-permissions-to-selected-namespaces-on-kubernetes.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Restrict permissions to selected namespaces on Kubernetes
```

Comment:
This isn't the guide we discussed no?

I thought we said:

1. install with restricted to just 1 ns
2. install the demo there
3. show it works
4. install the demo in another namespace
5. show it doesn't work
6. reinstall with the 2nd namespace added
7. restart the workload in the 2nd namespace
8. show it works

This ^ is most likely what users will actually do.

Reply:
done